internal/api/integrations/k8s/k8s.go

```diff
@@ -67,17 +96,46 @@ func (k *K8sClient) ListAll(namespace string) ([]models.Release, error) {
 				Sumologic: annotations.Sumologic(),
 			},
 			Code: models.SourceCode{
-				Github: annotations.Code(),
+				Github: gitRepo,
+				Ref:    tag,
 			},
 			Artifacts: models.Artifacts{
 				Chart: models.HelmArtifact{
 					Path:       r.Spec.Chart.Path,
 					Repository: r.Spec.Chart.Repository,
 					Version:    r.Spec.Chart.Revision,
 				},
+				Docker: models.DockerArtifact{
+					Image: repository,
```

Comment:
I think this field should be the absolute path to the image, the `image.URI()`, instead of just the repo, which would be something like `word-counts`.

Reply:
Should the `DockerArtifiact` type simply be removed then? If we give the full URI, other information such as the tag and repository name can be parsed out. Perhaps the `Docker` field could just be the URI string. For now I have put the full URI in the `Image` field as requested.